eth/rlp/transactions.py

```diff
@@ -83,6 +83,8 @@ def hash(self) -> bytes:
 
 
 class BaseTransaction(BaseTransactionFields, BaseTransactionMethods):
+    _sender_cache = None
```

Comment:
If this is a simple backing field, maybe just calling it `_sender` would be more inline with the rest of the code? When I read `cache` I think of a bag of things such as a `Dict[K,V]` or so.